frappe/core/doctype/doctype/doctype.js

```diff
@@ -55,5 +55,11 @@ frappe.ui.form.on('DocType', {
 
 	autoname(frm) {
 		frm.set_df_property('fields', 'reqd', frm.doc.autoname !== 'Prompt');
+	},
+
+	has_web_view(frm) {
+		if (!frm.doc.custom && frm.doc.has_web_view) {
+			frappe.msgprint(__("Check <b>Custom Field</b> in case of DocType without controllers"));
```

Comment:
no html tags inside translatable strings

Also make the message clear. Like "Please add a custom field for ____ since this is a custom web view"